setup.cfg

```diff
@@ -79,6 +79,7 @@ extend-ignore =
     F811,
     # line too long
     E501,
+    C408,
```

Comment:
Should: Why do we need this ignore? Others have a comment as to why (though I appreciate they're not great comments)

Reply:
It was to avoid the pre-commit checks from failing when dict() or tuple() is called instead of {}, (). 

It's from one of the first commits though and from running the pre-commits again it looks like most of those went away. Right now it only fails in one instance in this file `.github/pages/make_switcher.py`, so it would be easy enough to fix and remove the ignore from setup.cfg.